public/app/plugins/panel/stat/common.ts

```diff
@@ -85,6 +85,21 @@ export function addStandardDataReduceOptions<T extends SingleStatBaseOptions>(
       },
       defaultValue: '',
     });
+
+    builder.addRadio({
+      path: 'reduceOptions.sort',
+      name: 'Sort',
+      description: 'Sort by field display name',
+      category: valueOptionsCategory,
+      settings: {
+        options: [
+          { value: SortWithReducer.None, label: 'None' },
```

Comment:
The sort transformation doesn't categorize it like this, there's just the default and a reverse boolean. Maybe I should change to ascending / descending?